src/main/java/com/redhat/devtools/intellij/tektoncd/WindowToolFactory.java

```diff
@@ -48,6 +49,7 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
             tree.putClientProperty(Constants.STRUCTURE_PROPERTY, structure);
             tree.setCellRenderer(new NodeRenderer());
             tree.addTreeWillExpandListener(new TreeExpansionListener());
+            HubModel.getInstance();
```

Comment:
What is the rationale ? Is model is loaded asynchronously then `getInstance` should return a `CompletableFuture` and should called when required